src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -72,6 +72,27 @@ public void ProcessStart_UseShellExecute_OnLinux_ThrowsIfNoProgramInstalled()
             }
         }
 
+        [OuterLoop("Opens program")]
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Linux)]
+        public void ProcessStart_UseShellExecute_DirectoryAndFileWithSameName_Success()
+	{
+            string fileToOpen = "gobd";
```

Comment:
could contain text as little as:

#!/bin/sh
echo "believe me"

Reply:
remove gobd from path and run test again to make sure it still succeeds after fix.

reconfirm test fails before fix